pkg/controllers/provisioning/scheduling/nodeclaimtemplate.go

```diff
@@ -82,6 +82,7 @@ func (i *NodeClaimTemplate) ToNodeClaim(nodePool *v1beta1.NodePool) *v1beta1.Nod
 		},
 		Spec: i.Spec,
 	}
+	nc.Spec.StartupTaints = append(nc.Spec.StartupTaints, v1beta1.UnregisteredNoExecuteTaint)
```

Comment:
Do we need to add this into the spec if it's in our known ephemeral taints?

Reply:
No longer being added here.